src/redux/actions/samples/loadSamples.js

```diff
@@ -18,14 +18,18 @@ const loadSamples = (
     const response = await fetchAPI(url);
     const data = await response.json();
 
+    let samples;
+    if (!experimentId) samples = data[0].samples;
```

Comment:
The api should follow the same schema for the same request, why does it vary?

Reply:
API schema call for `projects/{projectId}/samples` : 

```
application/json:
  schema:
    type: array
    items:
      type: object
      properties:
        experimentId:
          type: string
        projectUuid:
          type: string
        samples:
          $ref: ./models/api-body-schemas/Samples.v1.yaml
```

API schema for `/experiments/{experimentId}/samples` :

```
application/json:
  schema:
    $ref: ./models/api-body-schemas/Samples.v1.yaml
```